test/unit/specs/table.spec.js

```diff
@@ -1758,6 +1758,52 @@ describe('Table', () => {
         });
       }, DELAY);
     });
+
+    it('sort correct change icon', done => {
```

Comment:
For asynchronous testing, use the async function.

Reply:
I found utils function `waitImmediate`. Good!